packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -231,7 +232,7 @@ export class FeatureDevController {
                     break
             }
         } catch (err: any) {
-            if (err instanceof ContentLengthError) {
+            if (err instanceof ToolkitError && err.code === 'ContentLengthError') {
```

Comment:
If we modify this check, how will the `ContentLengthError` be handled that is being thrown explicitly [here](https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/amazonqFeatureDev/client/featureDev.ts#L120) ?

Reply:
Good catch, I need to handle both.